src/components/Button.jsx

```diff
@@ -10,8 +10,8 @@ import './Button.css';
 class Button extends Component {
   render() {
     return (
-      <button className="button" onClick={() => this.props.handleClick()} name={this.props.number} type="submit">
-        {this.props.number}
+      <button className={this.props.className} onClick={this.props.onClick} type="submit">
+        {this.props.value}
       </button>
     );
   }

```

Comment:
- [ ] <b>REQUIRED:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.